internal/npm_package/packager.js

```diff
@@ -24,8 +24,12 @@ function mkdirp(p) {
   }
 }
 
-function write(p, content) {
+function write(p, content, replacements) {
   mkdirp(path.dirname(p));
+  for (let i = 0; i < replacements.length; i++) {
```

Comment:
Could do a ```replacements.forEach(r => {
   const [regexp, newvalue] = r;
   content = content.replace(regexp, newvalue);
})``` if you wanted to avoid the for loop

Reply:
or
```
for (let r of replacements) {
   const [regexp, newvalue] = r;
   content = content.replace(regexp, newvalue);
}
```
works too

Reply:
sure